sdk/nodejs/dynamodb/dynamodbMixins.ts

```diff
@@ -28,10 +28,38 @@ export interface TableEventSubscriptionArgs {
      */
     readonly batchSize?: number;
```

Comment:
Do we want to add these?

```
readonly destinationConfig?: pulumi.Input<inputs.lambda.EventSourceMappingDestinationConfig>;
readonly startingPositionTimestamp?: pulumi.Input<string>;
```

Reply:
starting position timestamp is only for kinesis

Reply:
Ah, okay. And `destinationConfig`?

Reply:
I DMed you in Slack :)